tests/Dockerfile_no_lock

```diff
@@ -0,0 +1,8 @@
+FROM python:3.7-slim-buster
+RUN pip install poetry
+COPY pyproject.toml /
+RUN poetry config virtualenvs.create false \
```

Comment:
I think there needs to be `RUN poetry lock` command here. To create a most up to date lock file from the config.

Reply:
`poetry install` creates a lock file if there isn't one, so shouldn't be necessary